assets/js/hooks/player-syncing.js

```diff
@@ -18,14 +18,18 @@ const updateVideoSlider = (
 }
 
 
-const udpateTimeDisplays = (startTimeTrackerElem, endTimeTrackerElem, timeSliderElem, player) => {
+const updateTimeDisplays = (startTimeTrackerElem, endTimeTrackerElem, timeSliderElem, player) => {
   const currentTime = player.getCurrentTime()
   const totalTime = player.getDuration()
   updateTimeDisplay(startTimeTrackerElem, currentTime)
   updateTimeDisplay(endTimeTrackerElem, totalTime)
   updateVideoSlider(timeSliderElem, currentTime, totalTime)
 }
 
+const playNextVideo = (hookContext) => {
+  hookContext.pushEvent('next_video')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dj-rumble/dj-rumble/pullRequest?prid=7333149)
```suggestion
  hookContext.pushEvent("next_video")
```